awx/main/models/jobs.py

```diff
@@ -545,6 +589,14 @@ def ansible_virtualenv_path(self):
     def event_class(self):
         return JobEvent
 
+    def copy_unified_job(self, **new_prompts):
+        # Needed for job slice relaunch consistency, do no re-spawn workflow job
+        # target same slice as original job
+        new_prompts['_prevent_slicing'] = True
+        new_prompts.setdefault('_eager_fields', {})
+        new_prompts['_eager_fields']['job_slice_number'] = self.job_slice_number
```

Comment:
add to this `job_slice_count`, related to https://github.com/ansible/awx/issues/2464 bug

will affect case of launching slice JT, changing the slice count, and then relaunching an old slice. We need to err on the side of the prior job values.

This may also resolve #2464 automatically ... but dunno how I feel about that.